src/main/resources/schemas/nhinc/common/NhincCommon.xsd

```diff
@@ -45,6 +45,7 @@
             <xsd:element name="userInfo" type="tns:UserType"/>
             <xsd:element name="authorized" type="xsd:boolean"/>
             <xsd:element name="purposeOfDisclosureCoded" type="tns:CeType" minOccurs="0"/>
+            <xsd:element name="samlConditions" type="tns:SamlConditionsType" minOccurs="0"/>
```

Comment:
Optional: Should this become samlCondition instead of samlConditions?  We only have a single condition in saml assertion